packages/expo-image/src/web/ColorTintFilter.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+export function getTintColorStyle(tintColor?: string | null) {
+  if (!tintColor) return {};
```

Comment:
```suggestion
  if (!tintColor) {
    return {};
  }
```
I know I'm annoying, but it looks better 😅